charts/canary/crds/http-monitor.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   # name must match the spec fields below, and be in the form: <plural>.<group>
   name: canaryhttpmonitors.canary.ukserp.ac.uk
+  namespace: default #TODO - template this
```

Comment:
I know we can't template in the `crds/` dir but if we leave the namespace undefined does the chart install it to `.Release.Namespace` ?